portlist/portlist_linux.go

```diff
@@ -27,6 +27,8 @@ import (
 
 func init() {
 	newOSImpl = newLinuxImpl
+	// Reading the sockfiles on Linux is very fast, so we can do it often.
```

Comment:
Fast might still cost a bunch of battery on laptops? Is the increased resolution worth it?